### PR TITLE
fix: hide upstream related btns if stack/series hasn't been pushed yet

### PR DESCRIPTION
**Pull request by ndom91**
## ☕️ Reasoning

- Hide upstream related btns on series header (i.e. "open in browser" and "create PR") if the stack hasn't been pushed yet.

### CODE REVIEW

File: apps/desktop/src/lib/branch/StackingBranchHeader.svelte [hunk 1]
```diff
@@ -135,33 +137,43 @@
...
 		</div>
-	</div>
+	{/if}
+	<PrDetailsModal
+		bind:this={prDetailsModal}
+		type="preview-series"
+		{upstreamName}
+		{name}
+		{commits}
+	/>
```
Comment (estib-vega) [inline apps/desktop/src/lib/branch/StackingBranchHeader.svelte:164]: It's probably not an actual problem, but I think the modal should be a outside the `div`

Comment (ndom91) [reply apps/desktop/src/lib/branch/StackingBranchHeader.svelte:164]: Yeah so it was originally, but I had to move it here to get the DOM structure to work for the css `:last-child`, for hiding the bottom border when appropriate. Putting the modal here shouldn't have any effect on its rendering, etc.

**Review by PavelLaptev — APPROVED** [review]

File: apps/desktop/src/lib/branch/StackingBranchHeader.svelte (hunk 1 above)
Comment (estib-vega) [reply apps/desktop/src/lib/branch/StackingBranchHeader.svelte:164]: Sounds good!

**Review by estib-vega — APPROVED** [review]
LGTM

**Pull request merged**